onboard/nav/searcher.cpp

```diff
@@ -268,6 +259,13 @@ NavState Searcher::executeDriveToBall( Rover * phoebe, const rapidjson::Document
     return NavState::TurnToBall;
 } // executeDriveToBall()
 
+bool Searcher::tennisBallReachable( Rover* phoebe, double distance, double bearing ) 
```

Comment:
This function is identical to one defined in `obstacleAvoidanceStateMachine.cpp`. We should put it somewhere that both these files have access to (e.g. `utilities.cpp`, `rover.cpp`, etc., wherever makes the most sense)

Reply:
They're not identical. The greater than, less than signs are different.